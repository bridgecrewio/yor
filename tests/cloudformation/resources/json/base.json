{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        },
        "Tags": [
          {
            "Key": "old_tag",
            "Value": "old_value"
          },
          {
            "Key": "yor_trace",
            "Value": "602154bb-d1f5-47a3-8d6e-3e0f2cd6c32d"
          }
        ]
      },
      "DeletionPolicy": "Retain"
    }
  }
}