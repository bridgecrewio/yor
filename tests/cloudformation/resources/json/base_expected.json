{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        },
        "Tags": [
          {
            "Key": "old_tag",
            "Value": "old_value"
          },
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          }
        ]
      },
      "DeletionPolicy": "Retain"
    }
  }
}
